ml-core-default/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply from: configs.androidLibrary
+
+dependencies {
+    api project(':ml-core-base')
+    api 'org.tensorflow:tensorflow-lite-support:0.4.3'
```

Comment:
The ml-core-default is built with portable tensorflow lite and tensorlfow lite-support library.

Note 1: we use `api` to "leak" the support library to downstream dependencies so that they don't need to explicitly declare it.

Note 2: atm only Identity is using support library to convert an image to an input tensor([source](https://github.com/stripe/stripe-android/blob/2b6460040ea3819af270d2bc8af0c52aba021a10/identity/src/main/java/com/stripe/android/identity/ml/IDDetectorAnalyzer.kt#L50)), it adds about 4kb to the overall download size. This is applicable to both `ml-core-default` and `ml-core-googleplay`